File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    let answer = [];
    const gcd = (a, b) => a % b === 0 ? b : gcd(b, a % b);
    const lcm = (a, b) => a * b / gcd(a, b);
    answer.push(gcd(n, m));
    answer.push(lcm(n, m));
    return answer;
}

File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def solution(n, m):
    answer = [gcd(n, m), lcm(n, m)]
    return answer

def gcd(a, b):
    if a % b == 0:
        return b
    return gcd(b, a % b)

def lcm(a, b):
    return a * b / gcd(a, b)